src/org/zaproxy/zap/extension/httpsessions/PopupMenuSessionSearch.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.httpsessions;
+
+import java.awt.Component;
+import java.util.regex.Pattern;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.ExtensionPopupMenuItem;
+
+public class PopupMenuSessionSearch extends ExtensionPopupMenuItem {
+
+	private static final long serialVersionUID = 1L;
+
+	public PopupMenuSessionSearch(final ExtensionHttpSessions extHttpSessions) {
+		super(Constant.messages.getString("httpsessions.popup.find"));
+		this.addActionListener(new java.awt.event.ActionListener() {
+
+			@Override
+			public void actionPerformed(java.awt.event.ActionEvent e) {
+				Pattern pattern = Pattern.compile(extHttpSessions.getHttpSessionsPanel().getSelectedSession()
+						.getTokenValuesString().replaceAll(";", "|"));
+				extHttpSessions.search(pattern);
+			}
+		});
+	}
+
+	@Override
+	public boolean isEnableForComponent(Component invoker) {
+		if (invoker.getName() != null && invoker.getName().equals(HttpSessionsPanel.PANEL_NAME)) {
+			this.setEnabled(true);

```

Comment:
It should be disabled if no session is selected, e.g. `PopupMenuItemCopySessionToken`. (The above condition could also be tweaked to not compare to null)